qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -207,12 +226,15 @@ def close_tab(self, tab):
         """
         last_close = config.get('tabs', 'last-close')
```

Comment:
I think it'd be cleaner to have one `if self.check_forms_can_be_destroyed(tab):` around the existing if-block here.